### PR TITLE
Update documentation based on comments in previous patch

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/lib/src/widgets/media_query.dart [hunk 1]
```diff
@@ -52,7 +52,11 @@ class MediaQuery extends InheritedWidget {
   /// The result of media queries in this subtree.
   final MediaQueryData data;
 
-  /// Queries the current media for the given context.
+  /// Returns the media query data for the given context.
+  ///
+  /// You can us this function to query the size an orientation of the screen.
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/media_query.dart]: us->use

File: packages/flutter/lib/src/widgets/media_query.dart [hunk 2]
```diff
@@ -52,7 +52,11 @@ class MediaQuery extends InheritedWidget {
...
+  /// When that information changes, your widget will be schedule to be rebuilt,
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/media_query.dart]: scheduled

File: packages/flutter/lib/src/widgets/overlay.dart [hunk 3]
```diff
@@ -110,8 +110,9 @@ class OverlayState extends State<Overlay> {
 
   /// (DEBUG ONLY) Check whether a given entry is visible (i.e., not behind an opaque entry).
   ///
-  /// This is an O(N) algorithm, and should not be necessary except for debug asserts.
-  /// To avoid people depending on it, we only implement it in checked mode.
+  /// This is an O(N) algorithm, and should not be necessary except for debug
+  /// asserts. To avoid people depending on it, this function is implmented only
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/overlay.dart]: implemented typo

File: packages/flutter/lib/src/widgets/pages.dart [hunk 4]
```diff
@@ -28,7 +28,7 @@ abstract class PageRoute<T> extends ModalRoute<T> {
     return performance;
   }
 
-  /// Subclasses can override this method to customize way heroes are inserted
+  /// Subclasses can override this method to customize way heroes are inserted.
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/pages.dart]: the grammar's still dodgy. :-)

> Subclasses can override this method to customize _the_ way heroes are inserted.
Or
> Subclasses can override this method to customize _how_ heroes are inserted.

**Comment by Hixie** [comment]
LGTM

**Pull request merged**